docusaurus/docs/guides/alt-server-certs.md

```diff
@@ -4,68 +4,75 @@ sidebar_label: Public CA Certs
 sidebar_position: 50
 ---
 
-You can configure the Ziti controller and routers to present alternative server certificates to Edge clients. This is useful in a deployment scenario where the Edge clients use ephemeral identities that can not be preconfigured to trust a particular certificate authority (CA) and must rely on the OS trust bundle of CA certificates.
+You can configure the Ziti controller and routers to present alternative server certificates, but it's not necessary in the most common cases. There are certain scenarios where a server certificate from a trusted third-party is necessary. 
 
-As a rule, Ziti Edge SDKs will insist that the server certificates they encounter are verifiable with the CA certificate bundle embedded in the identity configuration. If the CA bundle is absent, then the server certificates must be verifiable by the OS trust bundle. This means that it's necessary to configure all Ziti routers and the Ziti controller, and Ziti Edge SDKs in the same way, i.e., for alternative server certificates from a publicly-verifiable third-party CA like Let's Encrypt or with certificates that are verifiable with the trust bundle that the controller publishes, e.g., `https://ctrl.example.com:1280/.well-known/est/cacerts`.
+Alternative server certificates are most relevant to Ziti networks that use [BrowZer](https://blog.openziti.io/series/browzer). BrowZer is an implementation of Ziti's external ID claim feature. Unlike enrollment, which configures an explicit root of trust for mTLS, an external ID claim allows short-lived authentication instruments, e.g., JWT or x509 client cert, issued by a trusted identity provider to "claim" an existing Ziti identity. This is how a BrowZer user is able to connect to a Ziti service with an authorization token from an OIDC provider.
```

Comment:
`external id claim` is said here, but that term has no standard meaning and it doesn't link to anywhere to explain what it is.